src/components/navbar/Account.tsx

```diff
@@ -0,0 +1,27 @@
+import { AccentText } from 'components/Text'
+import ENSAddress from 'components/ENSAddress'
+import Network from 'models/Network'
+
+export default function ({
+  account,
+  needNetworkChange,
+}: {
+  account?: string
+  needNetworkChange: boolean
+}) {
+  const NotConnected = () =>
+    needNetworkChange ? <span>Change network</span> : <span>No wallet</span>
+
+  return (
+    <AccentText
+      extraSmall
+      color={account ? 'text-accent' : 'text-primary-semi-dimmed'}
```

Comment:
`text-primary-semi-dimmed` is default text colour, so you can try to avoid specifying it, but not sure how, if it's hard, leave as it is

Reply:
Nope, default color is `formal-accent` or so. That's why we need it here